src/System.Net.HttpListener/tests/HttpListenerResponseTests.Cookies.cs

```diff
@@ -170,6 +170,24 @@ public static IEnumerable<object[]> Cookies_TestData()
             Assert.Contains($"\r\nSet-Cookie: name3=value3\r\n", clientResponse);
             Assert.DoesNotContain("Set-Cookie2", clientResponse);
         }
+  
+        [Fact]
+        public async Task Cookies_AddMultipleInHeader_ClientReceivesExpectedHeaders()
+        {
+            HttpListenerResponse response = await GetResponse();
+            response.Headers.Add("Set-Cookie", "name1=value1");
+            response.Headers.Add("Set-Cookie", "name2=value2");
+            response.Headers.Add("Set-Cookie", "name3=value3");
+            response.Headers.Add("Set-Cookie", "name4=value4");
+
+            response.Close();
+
+            string clientResponse = GetClientResponse(224);
```

Comment:
What is "224"?  Perhaps using named parameters in the call site here or adding a comment would help.

Reply:
It's the size of the expected message, and is expected by this somewhat unwieldy helper method:
https://github.com/dotnet/corefx/blob/fb0cbae3e0275fdf9b2b7bc734422c88285acafb/src/System.Net.HttpListener/tests/HttpListenerResponseTests.cs#L34-L50
Using a named parameter would definitely make things clearer, so I'll add that,